### PR TITLE
Clean up Fingerprint Data to spaced string methods

### PR DESCRIPTION
**Pull request by billypchan**
## What's new in this PR?

Commented by @alexaubry in https://github.com/wireapp/wire-ios/pull/3463#discussion_r292808013, unify the Fingerprint Data to spaced string methods with existing `NSString.fingerprintStringWithSpaces()` method.

### CODE REVIEW

File: Wire-iOS/Sources/Helpers/NSData+Fingerprint.swift [hunk 1]
```diff
@@ -1,63 +1,60 @@
...
+//
 
 
 import Foundation
 
-extension NSData {
+extension Data {
+    /// return a lower case and space between every btye string of the given data
+    var fingerprintString: String? {
+        let string = String(data: self, encoding: .utf8)
```
Comment (alexaubry) [inline Wire-iOS/Sources/Helpers/NSData+Fingerprint.swift]: You can use the non nullable decoder here:

```suggestion
        let string = String(decoding: self, as: UTF8.self)
```

File: Wire-iOS/Sources/Helpers/NSData+Fingerprint.swift [hunk 2]
```diff
@@ -1,63 +1,60 @@
...
+
         let paragraphStyle = NSMutableParagraphStyle()
         paragraphStyle.lineSpacing = 10
         return attributedRemoteIdentifier! && [.paragraphStyle: paragraphStyle]
     }
-    
+}
+
+extension NSData {
     public func mapBytes<T: Any, E: Any>(callback: (E) -> (T)) -> [T] {
```
Comment (typfel) [inline Wire-iOS/Sources/Helpers/NSData+Fingerprint.swift]: Is `mapBytes` still in used anywhere else?

Comment (billypchan) [reply Wire-iOS/Sources/Helpers/NSData+Fingerprint.swift]: removed!

File: Wire-iOS/Sources/Helpers/NSData+Fingerprint.swift [hunk 3]
```diff
@@ -1,79 +1,44 @@
...
 // along with this program. If not, see http://www.gnu.org/licenses/.
-// 
+//
 
 
 import Foundation
 
-extension NSData {
+extension Data {
+    /// return a lower case and space between every btye string of the given data
```
Comment (typfel) [inline Wire-iOS/Sources/Helpers/NSData+Fingerprint.swift]: Typo

```suggestion
    /// return a lower case and space between every byte string of the given data
```

**Pull request merged**